### PR TITLE
Package highfive

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: recipes/highfive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - hdf5
+    - boost
+    - boost-cpp
+  run:
+    - hdf5
```
Comment (bgruening) [inline recipes/highfive/meta.yaml]: It seems that run_exports are already defined: https://github.com/conda-forge/hdf5-feedstock/blob/master/recipe/meta.yaml#L62

So you don't need to specify it in your run requirements.

File: recipes/highfive/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  number: 0
+  skip: true  # [win and py<36]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - hdf5
+    - boost
```
Comment (bgruening) [inline recipes/highfive/meta.yaml]: Are you sure you need `boost`? If so, then `boost-cpp` can be removed afaik.

File: recipes/highfive/build.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DHIGHFIVE_UNIT_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib $SRC_DIR
+make install
```
Comment (bgruening) [inline recipes/highfive/build.sh]: Is there something like `make test`?

File: recipes/highfive/meta.yaml (hunk 1 above)
Comment (SylvainCorlay) [reply recipes/highfive/meta.yaml]: Thanks, removed the run requirement.

File: recipes/highfive/meta.yaml (hunk 2 above)
Comment (SylvainCorlay) [reply recipes/highfive/meta.yaml]: it seems so, removed boost-cpp

**Comment by SylvainCorlay** [comment]
The testing issues appear to be related to https://github.com/conda-forge/hdf5-feedstock/issues/58

**Comment by SylvainCorlay** [comment]
Skipping windows for now. Will be doing the windows packaging in a PR.

File: recipes/highfive/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  license_file: LICENSE
+  summary: 'Header-only C++ HDF5 interface'
+  description: 'HighFive is a modern C++/C++11 friendly interface for libhdf5.'
+  dev_url: https://github.com/BlueBrain/HighFive
+
+extra:
+  recipe-maintainers:
+    - wolfv
+    - SylvainCorlay
+    - JohanMabille
```
Comment (scopatz) [inline recipes/highfive/meta.yaml]: @wolfv @JohanMabille - are you willing to be maintainers here?

Comment (scopatz) [reply recipes/highfive/meta.yaml]: You can remove yourselves in the feedstock if you want :wink:

Comment (SylvainCorlay) [reply recipes/highfive/meta.yaml]: @wolfv @JohanMabille no way :smirk:

**Pull request merged**